server.js

```diff
@@ -3,19 +3,105 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+//depolyed at Heruku: https://annatereliusmongoapi.herokuapp.com/
+
+// WAfV2IEidqVDmDR5 password mongo cloud
+// mongodb+srv://annaholly:<password>@cluster0-trdcw.mongodb.net/test?retryWrites=true&w=majority connectionstring
+// connectionstring Heruku/mongocloud : mongodb+srv://annaholly:WAfV2IEidqVDmDR5@cluster0-trdcw.mongodb.net/mongo_api?retryWrites=true&w=majority
+//PMAK-5e3c53897371d60031f47163-12e3caaadf13341996d81dc084250d29d3 api-key for postman
```

Comment:
Usually a good rule of thumb to not commit passwords/etc to git, for future reference 😄